ci/ansible/roles/pulp-build/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Get user's home directory
+  getent:
+    database: passwd
+    key: '{{ ansible_user }}'
+    split: ':'
+
+- name: Set variable referencing user's home directory
+  set_fact:
+    ansible_user_home: "{{ getent_passwd[ansible_user][4] }}"
+
+- name: Export packer template
+  template:
+    src: pulpcore.json.jj2
+    dest: "{{ ansible_user_home }}/pulpcore.json"
+
+- name: Export Vagrantfile template
+  template:
+    src: Vagrantfile.jj2
+    dest: "{{ ansible_user_home }}/Vagrantfile"
+
+- name: Download installation playbook
+  get_url:
+    url: https://raw.githubusercontent.com/pulp/devel/3.0-dev/ansible/deploy-pulp3.yml
+    dest: "{{ ansible_user_home }}"
+
+- name: Download requirements.yml
+  get_url:
+    url: https://raw.githubusercontent.com/pulp/devel/3.0-dev/ansible/requirements.yml
+    dest: "{{ ansible_user_home }}"
+
+- name: Install ansible
+  package:
+    name: ansible
+    state: installed
+
+- name: Install ansible galaxy roles
+  command: ansible-galaxy install -r requirements.yml
+
+- name: Reboot Server
```

Comment:
I added this, because at the end of a build, both VirtualBox and KVM are running, hence, on the next build, KVM build fails saying "Device or Resource is busy". This is because, KVM can't take advantage of VT-x, while VirtualBox is running.